### PR TITLE
UA for Infrastructure - Update /support/plans-and-pricing page

### LINKED ISSUE #4704: UA for infrastructure - Update support/plans-and-pricing
Based on the [copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#heading=h.xv4ec899rw2v)

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Updated the plans-and-pricing page for UA for infrastructure
- Updated the includes that appear in openstack and kubernetes that relate to this page
- Removed some unused includes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#)

## Issue / Card

Fixes #4704

### CODE REVIEW

File: templates/shared/pricing/_ua-for-infrastructure-physical.html [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+        <tr>
+          <td>High availability support (Kubernetes and Openstack)</td>
+          <td>&nbsp;</td>
+          <td>&nbsp;</td>
+          <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" width="16" height="16" alt="Yes" /></td>
+        </tr>
+
+        <tr>
+          <td>24/7 Openstack Support</td>
```
Comment (cjohnston1158) [inline templates/shared/pricing/_ua-for-infrastructure-physical.html]: Should these be "OpenStack"?

File: templates/shared/pricing/_ua-for-infrastructure-physical.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+        <tr>
+          <td>Storage Allowance for Ceph/Swift <a href="#fn-ua-i-storage"><sup>*</sup></a></td>
+          <td>&nbsp;</td>
+          <td>&nbsp;</td>
+          <td class="u-align--center"><img src="{{ ASSET_SERVER_URL }}f1a7515d-tick-darkaubergine.svg" width="16" height="16" alt="Yes" /></td>
+        </tr>
+
+        <tr>
+          <td>High availability support (Kubernetes and Openstack)</td>
```
Comment (anthonydillon) [inline templates/shared/pricing/_ua-for-infrastructure-physical.html]: OpenStack here too

**Comment by anasereijo** [comment]
please fix:

plans and pricing support ubuntu

**Comment by anasereijo** [comment]
👍 LGTM

**Pull request merged**